src/detectors/HCAL/HCAL.cc

```diff
@@ -45,7 +45,6 @@ extern "C" {
 
         app->Add(new JFactoryGeneratorT<RawCalorimeterHit_factory_HcalBarrelRawHits>());
         app->Add(new JFactoryGeneratorT<CalorimeterHit_factory_HcalBarrelRecHits>());
-        app->Add(new JFactoryGeneratorT<CalorimeterHit_factory_HcalBarrelMergedHits>());
```

Comment:
This warrants removal of a corresponding file.

Reply:
Ah! Would the best practice be to delete the corresponding .h file, or to leave the .h file and this line in?

Reply:
Yes, delete the .h and .cc files with the corresponding factory.

Normally, you'd need to also remove a `"HcalBarrelMergedHits"` line in
https://github.com/eic/EICrecon/blob/1f30d3b5e2aef6fdc9e7bccfe8b4e1228f63b551/src/services/io/podio/JEventProcessorPODIO.cc#L167-L179
but it seems like they forgot to add it in the first place.

Reply:
Gotcha! I'll remove those!